woocommerce/woocommerce-android
# Create base address form

Summary
==========
Fixes issue #4893 and partially delivers issues #4978, #4894 and #4901 by creating an abstract Address Form view attached to the `BaseOrderEditingFragment` structure and creating the base scaffold for the Shipping and Billing Address editing views. The objective of this view is to 

⚠️ There are a lot of improvements pending that will be delivered in upcoming PRs, here they are:
- Make `BaseOrderEditingFragment` extendable for any part of the app, not only Order Editing.
- Detect & Submit editing changes
- Country & State selection views
- Shipping & Billing specific actions

Screenshots
==========
| Order Details  | Address form |
| ------------- | ------------- |
| ![Screenshot_20211011_185413](https://user-images.githubusercontent.com/5920403/136860901-7b1d990a-a26b-4edd-92e6-8bb0a7e7395b.png) | ![Screenshot_20211011_185422](https://user-images.githubusercontent.com/5920403/136860915-89a7660c-0089-4159-a5d9-367045ac63a9.png) |

How to Test
==========
1. Go to the Order Details of an Order containing at least the Billing Address filled
2. Select the pencil icon of either the Shipping details or Billing details
3. Verify if the Address Form opens displaying the expected data for each section
4. Interact with the fields and see if everything is editable and Forward Focus works as expected


Update release notes:
